README.md

```diff
@@ -17,6 +17,14 @@ Maintainer of coq-bench is Emilio Jesús Gallego Arias; see the
 [contributing](./CONTRIBUTING.md) file information about contributing
 to this software.
 
+## Adding your package for testing.
+
+- Add your package to the `coq-extra-dev` OPAM repository
+  https://github.com/coq/opam-coq-archive
+- Go to the web UI
+  https://ci.inria.fr/coq/view/benchmarking/job/benchmark-part-of-the-branch/configure
```

Comment:
Maybe this should explain that you need some permissions? ala https://github.com/coq/coq/wiki/Jenkins-(automated-benchmarking)#i-dont-have-a-jenkins-account

Reply:
Good point, updated.